src/js/apps/patients/worklist/worklist_app.js

```diff
@@ -292,6 +296,7 @@ export default App.extend({
 
     this.listenTo(typeToggleView, 'toggle:listType', listType => {
       this.setState('listType', listType);
+      this.setState('lastSelectedId', null);
```

Comment:
Clears `state.lastSelectedId` when the user toggles between the `Actions` and `Flows` versions of the page via the buttons below.
<img width="243" alt="Screen Shot 2022-07-18 at 5 12 55 PM" src="https://user-images.githubusercontent.com/35355575/179626165-001d73b4-738b-4cd4-86d6-09e490f541aa.png">

Reply:
Agreed, much cleaner.